documentation/domains/Cluster.md

```diff
@@ -66,7 +66,7 @@ The specification of the operation of the WebLogic cluster. Required.
 | `maxReadyWaitTimeSeconds` | integer | The maximum time in seconds that the operator waits for a WebLogic Server pod to reach the ready state before it considers the pod failed. Defaults to 1800 seconds. |
 | `nodeName` | string | NodeName is a request to schedule this Pod onto a specific Node. If it is non-empty, the scheduler simply schedules this pod onto that node, assuming that it fits the resource requirements. See `kubectl explain pods.spec.nodeName`. |
 | `nodeSelector` | Map | Selector which must match a Node's labels for the Pod to be scheduled on that Node. See `kubectl explain pods.spec.nodeSelector`. |
-| `podSecurityContext` | [Pod Security Context](k8s1.13.5.md#pod-security-context) | Pod-level security attributes. See `kubectl explain pods.spec.securityContext`. |
+| `podSecurityContext` | [Pod Security Context](k8s1.13.5.md#pod-security-context) | Pod-level security attributes. See `kubectl explain pods.spec.securityContext`. Beginning with operator version 4.0.5, the operator will use default content for the pod-level `securityContext` if no value is specified for this field. More info: https://oracle.github.io/weblogic-kubernetes-operator/security/domain-security/pod-and-container/ |
```

Comment:
-> Beginning with operator version 4.0.5, if no value is specified for this field, the operator will use default content for the pod-level `securityContext`.
-> Period at the end of the last sentence.